pyvista/utilities/common.py

```diff
@@ -0,0 +1,46 @@
+"""Common functions."""
+import collections.abc
+from typing import Sequence, Union
+
+import numpy as np
+
+PointLike = Union[Sequence, np.ndarray]
```

Comment:
Should we call this `ArrayLike` and just swap it with numpy's version when we can depend on >=1.22? Although I guess that would affect imports elsewhere too...

Reply:
This makes sense, but then it is not general which would argue for not having it in this module.  After having written out my thoughts, let's do that instead of my stop gap comment which I added recently.

Reply:
There is a _typing module, so it makes sense to have it there.  A lot of those types could be replaced by ArrayLike.  So I'm not sure that it makes sense to use it here.  Also, I changed to use the `Number` type for `PointsLike`

Reply:
Hmm, good point, it might not be worthwhile to be this specific with the typing. `NumericArray` might fit the bill. I'm good either way. But yeah, if we keep `PointsLike` it should be in `_typing`, good catch.

Reply:
Good point.  I've removed `PointsLike` and now use `Union[NumericArray, VectorArray]`.

I'm not sure if it is easier or more precise to type with the newer versions of numpy, but I hope so.